lib/admin/WP_Auth0_Admin_Appearance.php

```diff
@@ -85,16 +111,117 @@ public function init() {
 				'id'       => 'wpa0_language',
 				'function' => 'render_language',
 			),
-			array(
+		);
+
+		// TODO: Remove this once feature has been removed
+		if ( $this->options->get( 'language_dictionary' ) ) {
+			$options[] = array(
 				'name'     => __( 'Language Dictionary', 'wp-auth0' ),
 				'opt'      => 'language_dictionary',
 				'id'       => 'wpa0_language_dictionary',
 				'function' => 'render_language_dictionary',
+			);
+		}
+
+		$options = $options + array(
+			( count( $options ) ) => array(
+				'name'     => __( 'Custom Signup Fields', 'wp-auth0' ),
+				'opt'      => 'custom_signup_fields',
+				'id'       => 'wpa0_custom_signup_fields',
+				'function' => 'render_custom_signup_fields',
+			),
+			array(
+				'name'     => __( 'Extra Settings', 'wp-auth0' ),
+				'opt'      => 'extra_conf',
+				'id'       => 'wpa0_extra_conf',
+				'function' => 'render_extra_conf',
 			),
 		);
 		$this->init_option_section( '', 'appearance', $options );
 	}
 
+	/**
```

Comment:
All of these methods existed in the plugin already (see deprecated methods above).